Krause-House/houseOS
# add yarn prep command as PR checklist

### Problem
> What problem does this solve?  Roughly how urgent/severe is this?
1. Too much friction to navigate to a contributor guidelines page.  This will help remind devs and get muscle memory with a command they can run in under 2 seconds.


### Test Plan
> Did you add unit tests?  Does this require any manual review?  What should reviewers do to verify this code?
1. Run `yarn prep`
2. Ensure message is useful to you, if not, leave feedback in comments.


### Checklist (please confirm by adding x in brackets)
- [x] Relevant unit tests written if any (N/A)
- [x] Commit(s) do one thing and one thing only.
